x/dsmr/node.go

```diff
@@ -118,9 +126,9 @@ func (n *Node[T]) BuildChunk(
 	txs []T,
 	expiry int64,
 	beneficiary codec.Address,
-) (Chunk[T], ChunkCertificate, error) {
```

Comment:
This was so gross previously (and I'm to blame...), we previously exposed the chunk/chunk certs that were created into the caller. Realistically this is just leaking implementation details because hypersdk would never care about the chunks that were built - since the mempool/consensus only cares about blocks.